cmake/Install.cmake

```diff
@@ -292,23 +302,72 @@ configure_file("${CMAKE_SOURCE_DIR}/cmake/CMakeCPackOptions.cmake.in"
-  install(FILES "${CMAKE_BINARY_DIR}/doc-pdf/PlantApplicationGuide.pdf" DESTINATION "./Documentation")
-  install(FILES "${CMAKE_BINARY_DIR}/doc-pdf/TipsAndTricksUsingEnergyPlus.pdf" DESTINATION "./Documentation")
-  install(FILES "${CMAKE_BINARY_DIR}/doc-pdf/UsingEnergyPlusForCompliance.pdf" DESTINATION "./Documentation")
+  install(FILES "${CMAKE_BINARY_DIR}/doc-pdf/Acknowledgments.pdf" DESTINATION "./Documentation" COMPONENT Documentation)
+  install(FILES "${CMAKE_BINARY_DIR}/doc-pdf/AuxiliaryPrograms.pdf" DESTINATION "./Documentation" COMPONENT Documentation)
+  install(FILES "${CMAKE_BINARY_DIR}/doc-pdf/EMSApplicationGuide.pdf" DESTINATION "./Documentation" COMPONENT Documentation)
+  install(FILES "${CMAKE_BINARY_DIR}/doc-pdf/EngineeringReference.pdf" DESTINATION "./Documentation" COMPONENT Documentation)
+  install(FILES "${CMAKE_BINARY_DIR}/doc-pdf/EnergyPlusEssentials.pdf" DESTINATION "./Documentation" COMPONENT Documentation)
+  install(FILES "${CMAKE_BINARY_DIR}/doc-pdf/ExternalInterfacesApplicationGuide.pdf" DESTINATION "./Documentation" COMPONENT Documentation)
+  install(FILES "${CMAKE_BINARY_DIR}/doc-pdf/GettingStarted.pdf" DESTINATION "./Documentation" COMPONENT Documentation)
+  install(FILES "${CMAKE_BINARY_DIR}/doc-pdf/InputOutputReference.pdf" DESTINATION "./Documentation" COMPONENT Documentation)
+  install(FILES "${CMAKE_BINARY_DIR}/doc-pdf/InterfaceDeveloper.pdf" DESTINATION "./Documentation" COMPONENT Documentation)
+  install(FILES "${CMAKE_BINARY_DIR}/doc-pdf/ModuleDeveloper.pdf" DESTINATION "./Documentation" COMPONENT Documentation)
+  install(FILES "${CMAKE_BINARY_DIR}/doc-pdf/OutputDetailsAndExamples.pdf" DESTINATION "./Documentation" COMPONENT Documentation)
+  install(FILES "${CMAKE_BINARY_DIR}/doc-pdf/PlantApplicationGuide.pdf" DESTINATION "./Documentation" COMPONENT Documentation)
+  install(FILES "${CMAKE_BINARY_DIR}/doc-pdf/TipsAndTricksUsingEnergyPlus.pdf" DESTINATION "./Documentation" COMPONENT Documentation)
+  install(FILES "${CMAKE_BINARY_DIR}/doc-pdf/UsingEnergyPlusForCompliance.pdf" DESTINATION "./Documentation" COMPONENT Documentation)
 endif ()
 
-INCLUDE(CPack)
+include(CPack)
+include(CPackIFW)
+
+#cpack_add_component(EnergyPlus
+  #DISPLAY_NAME "EnergyPlus"
+  #DESCRIPTION "The EnergyPlus program itself"
+  #REQUIRED
+#)
+
+#cpack_add_component(AuxiliaryPrograms
+  #DISPLAY_NAME "Auxiliary Programs"
+  #DESCRIPTION "The suite of Fortran auxiliary programs such as ReadVarsESO, ExpandObjects, etc"
+  #REQUIRED
+#)
+
+cpack_add_component(Documentation
+  DISPLAY_NAME "Documentation"
+  DESCRIPTION "EnergyPlus documentation in PDF format"
+)
+
+cpack_add_component(Datasets
+  DISPLAY_NAME "Datasets"
+  DESCRIPTION "Useful resources such as material and equipment performance data"
+)
+
+cpack_add_component(ExampleFiles
+  DISPLAY_NAME "Example Files"
+  DESCRIPTION "IDF Example Files"
+)
+
+cpack_add_component(WeatherData
+  DISPLAY_NAME "Weather Data"
+  DESCRIPTION "A few EPW files"
```

Comment:
I'm going to make a change here to "A set of EPW files" rather than "A few..."

Reply:
Maybe "EPW Weather Files"? to mimic "IDF Example Files"

Reply:
Wouldn't that expand to "EnergyPlus Weather Weather Files"?

Reply:
@jasondegraw EPW is actually a recursive acronym where the E stands for EnergyPlus Weather Data Files, so the full name is "EnergyPlus Weather Data Files Plus Weather Weather Files ... the 3rd Baron of Townsville"

As bad as having a PIN number or a VIN number or and IDF file is, I think "EPW Weather Files" will make perfect sense to the users.

Reply:
Well, yes. Using that same logic it would be "Input Data File Examples" and "EnergyPlus Weather Files".